File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.py
def solution(price):
    answer = 0
    sale = 0
    if price >= 500000:
        sale = 20
    elif price >= 300000:
        sale = 10
    elif price >= 100000:
        sale = 5
    else:
        pass
    print(sale)
    answer = int(price - (price * (sale/100)))
    return answer

File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.java
class Solution {
    public int solution(double price) {
        double discount = 0.00;
        
        if(price>=100000 && price<300000){
            discount = 0.05;
        }
        else if(price<500000 && price>= 300000){
            discount = 0.10;
        }
        else if(price>= 500000){
            discount = 0.20;
        }
        
        
        int answer = (int)(price-(price*discount));
        return answer;
    }
}